example/patches/react-native-vision-camera+2.16.7.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/node_modules/react-native-vision-camera/android/build.gradle b/node_modules/react-native-vision-camera/android/build.gradle
```

Comment:
Have any context on this patch? At first I thought it was just moving boost to archives, but new checks in here for RN71 etc

Reply:
`node_modules/react-native/ReactAndroid/gradle.properties` no longer host values such as `BOOST_VERSION` etc..

this will always result to build errors since the task `downloadBoost` is trying to replace a `null or undefined` value 

checked the code and it has a comment that its mostly a copy of 
https://github.com/software-mansion/react-native-reanimated/blob/master/android/build.gradle

so I just added the same condition to only download boost if less than RN 71